libs/langgraph/langgraph/prebuilt/chat_agent_executor.py

```diff
@@ -161,6 +161,37 @@ def _should_bind_tools(model: LanguageModelLike, tools: Sequence[BaseTool]) -> b
     return False
 
 
+def _validate_messages(
```

Comment:
Maybe more generic name? we're not validating the messages individually, but the sequence of messages / history

```suggestion
def _validate_chat_history(
```

Reply:
nice, will rename